docs/Community/conduct/IncidentResponseGuidelines.md

```diff
@@ -0,0 +1,7 @@
+Incident Response Policy and Guidelines
+========================
+
+This document should outline the policy and guidelines which the recipient of a
+Conduct Incident report will follow.
+
+TODO
```

Comment:
What a big TODO 😅 There is some helpful guidance in https://www.contributor-covenant.org/faq/ and in https://opensource.guide/code-of-conduct/